src/spatialCursor/index.js

```diff
@@ -4,10 +4,10 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
 
 (function(exports) {
 
-    const SNAP_CURSOR_TO_TOOLS = true;
-    const DEFAULT_SPATIAL_CURSOR_ON = true;
+    const SNAP_CURSOR_TO_TOOLS = false; // the snapping doesn't do anything yet, so turn off for now
+    const DEFAULT_SPATIAL_CURSOR_ON = true; // applied after we localize within a target / download occlusion mesh
 
-    let isCursorEnabled = DEFAULT_SPATIAL_CURSOR_ON;
+    let isCursorEnabled = false; // always starts off, turns on after localize
```

Comment:
This flag is not used anywhere else. Did you mean to set it to true in the onLoadOcclusionObject callback?

Reply:
Good thought, but it gets set true in the `toggleDisplaySpatialCursor` function which onLoadOcclusionObject's callback calls. (You might need to unfold some hidden parts of the code in the code review page to see the full file)

Reply:
Got it!